README.md

```diff
@@ -0,0 +1,74 @@
+[![PyPI](https://badge.fury.io/py/mssql-cli.svg)](https://pypi.python.org/pypi/mssql-cli)
+[![Python 2,3](https://img.shields.io/badge/python-2.7,&nbsp;3.x-blue.svg)](https://github.com/dbcli/mssql-cli)
```

Comment:
I think we should be explicit about which python versions we support and avoid 3.x.